flo-freezer/src/main/java/com/spotify/flo/freezer/PersistingContext.java

```diff
@@ -52,6 +52,11 @@
  */
 public class PersistingContext extends ForwardingEvalContext {
 
+  static {
+    // Best effort. Hope that ObjectOutputStream has not been loaded yet :pray:
+    System.setProperty("sun.io.serialization.extendedDebugInfo", "true");
```

Comment:
Can we add this option when we do forking?

Reply:
Done.